cardano-integration/src/Test/Integration/Cardano.hs

```diff
@@ -1,8 +1,8 @@
-module Test.Integration.Cardano
-  ( module Test.Integration.Cardano.Local
-  , module Test.Integration.Cardano.Process
-  , module Test.Integration.Workspace
-  ) where
+module Test.Integration.Cardano (
+  module Test.Integration.Cardano.Local,
```

Comment:
FYI this style is called "diff-friendly" because it causes the fewest conflicts and has clean diffs when imports/exports are added or removed anywhere in the list